tensorflow/inference/docker/2.14/py3/cu118/Dockerfile.sagemaker.gpu.core_packages.json

```diff
@@ -0,0 +1,21 @@
+{
+    "protobuf":{
+        "version_specifier":">=4.25.3,<5.0"
+    },
+    "cython":{
+        "version_specifier":"<3.0"
+    },
+    "awscli":{
+        "version_specifier":"<2"
+    },
+    "tensorflow-serving-api-gpu":{
+        "version_specifier":"==2.14.1",
+        "skip":"True"
+    },
+    "falcon":{
+        "version_specifier":"==3.1.0"
```

Comment:
This should be `3.0.1`: https://github.com/aws/deep-learning-containers/blob/master/tensorflow/inference/docker/2.14/py3/cu118/Dockerfile.gpu#L227